nservicebus/rabbitmq/delayed-delivery.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Delayed Delivery
+summary: Describes the native delayed delivery implementation in the RabbitMQ transport
+component: Rabbit
+versions: '[4,]'
+---
+
+Starting with Version 4.3, the transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). The transport now creates an infrastructure inside the broker to natively handle delaying messages.
+
+
+## How it works
+
+The transport sets up a series of exchanges, queues, and bindings that work together to provide the necessary infrastructure to support delayed messages. These exchange & queues are grouped into a series of levels, 0 - 27. There is one final "delivery" exchange in addition to the level.
+
+
+### Delay levels
+
+Each level represents a delay amount corresponding to a number of seconds equal to 2 ^ level seconds. This is achieved by declaring the queue for each level with an `x-message-ttl` value corresponding to level's delay amount. It is also declared with an `x-dead-letter-exchange` value corresponding to the next lowest level. This means a message will sit in the queue for the amount of time specified in  `x-message-ttl` and then be routed to the next level.
```

Comment:
> "level's delay amount in seconds"?

It should actually be milliseconds here.

> Is that right? It's always points to the "the next lower level, not lowest. I.e. from level X to level X-1

Reading up on usage of "next lowest" it seems like my intended usage here is fairly common, but it can be rather ambiguous. I'll reword it to avoid the phrase altogether.